terraform/aws/modules/route53-registration/route53.tf

```diff
@@ -1,17 +1,26 @@
 data "kubernetes_service" "nginx-internal" {
   metadata {
-    name = "mattermost-cm-nginx-internal-nginx-ingress-controller"
+    name      = "mattermost-cm-nginx-internal-nginx-ingress-controller"
     namespace = "network"
   }
 }
 
-data "kubernetes_service" "nginx" {
+data "kubernetes_service" "nginx-public" {
   metadata {
-    name = "mattermost-cm-nginx-nginx-ingress-controller"
+    name      = "mattermost-cm-nginx-public-nginx-ingress-controller"
     namespace = "network"
   }
 }
 
+data "aws_elb_hosted_zone_id" "main" {}
+
+# data "kubernetes_service" "nginx" {
```

Comment:
@angeloskyratzakos Let clean this if not needed

Reply:
Just wanted to ask you if it was there on purpose. Will remove it.